src/lib/y2storage/filesystems/base.rb

```diff
@@ -65,6 +82,17 @@ def in_network?
       def types_for_is
         super << :filesystem
       end
+
+      FREE_SPACE_FALLBACK = DiskSize.new(0)
+      def detect_free_space
+        return FREE_SPACE_FALLBACK unless is?(:blk_filesystem)
+
+        size = blk_devices.map(&:size).reduce(:+)
+        used = detect_resize_info.min_size
```

Comment:
You could use `resize_info` instead of `detect_resize_info` here. In some scenarios it may be faster:
https://github.com/yast/yast-storage-ng/blob/97207385177e05fcd46afe48e66bfbc58c07391b/src/lib/y2storage/device.rb#L242